plugin/trino-pinot/src/test/java/io/trino/plugin/pinot/AbstractPinotIntegrationSmokeTest.java

```diff
@@ -1138,6 +1158,28 @@ public void testNullBehavior()
     {
         // Verify the null behavior of pinot:
 
+        // Default null value for timestamp single value columns is 0
+        assertThat(query("SELECT json_col" +
+                "  FROM " + ALL_TYPES_TABLE +
+                "  WHERE string_col = 'null'"))
+                .matches("VALUES(VARCHAR 'null')")
+                .isFullyPushedDown();
+
+        // Default null value for timestamp single value columns is 0
+        assertThat(query("SELECT timestamp_col" +
+                "  FROM " + ALL_TYPES_TABLE +
+                "  WHERE string_col = 'null'"))
+                .matches("VALUES(TIMESTAMP '1970-01-01 00:00:00.000')")
+                .isFullyPushedDown();
+
```

Comment:
Can you also add a test for timestamp predicate and pushdown?
E.g. `where timestamp_col > timestamp '2022-01-01 00:00:00.000'`

Reply:
@xiangfu0 thanks for suggesting this test! I found I had to add one more conversion